### DISCUSSION TITLE
Improvement of download menu

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#398] [Q&A] [answered]
- Combining POI and Routing download with map download
- Hiding POI download completely
- Improvement of 3 download tabs

**Comment by devemux86** [comment, marked as answer]
> Combining POI and Routing download with map download

This is already the case with Mapsforge maps, they are downloaded together with their poi + routing data.

Important for the routing data is that we need the country boundaries,
easily provided with Mapsforge maps.
